src/data/mv_polynomial/equiv.lean

```diff
@@ -220,6 +228,28 @@ def fin_succ_equiv (n : ℕ) :
 (ring_equiv_of_equiv α (fin_succ_equiv n)).trans
   (option_equiv_left α (fin n))
 
+@[simp] lemma fin_succ_equiv_apply (n : ℕ) (p : mv_polynomial (fin (n + 1)) α) :
+  fin_succ_equiv α n p =
+  eval₂_hom (polynomial.C.comp (C : α →+* mv_polynomial (fin n) α))
+    (λ i : fin (n+1), fin.cases polynomial.X (λ k, polynomial.C (X k)) i) p :=
+begin
```

Comment:
Do we have a specialized `ext` lemma for `ring_hom`s from `mv_polynomial`s? AFAIR, I wrote it for a `monoid_algebra` but probably failed to get it merged.

Reply:
We do. I pushed a new proof. It wasn't that much shorter unfortunately...
If these equivs were alg_hom equivs... then we could skip the C-goal